config.py

```diff
@@ -1,4 +1,11 @@
-DATABASE_URI = 'sqlite:///customer-support.db'
+import os
+
+if('ENV' in os.environ and os.environ['ENV'] == 'production'):
+	print('Using Mysql database')
+	DATABASE_URI = 'mysql+pymysql://customersupport:nAbUfr7juTHay@S=@localhost/customer-support'
```

Comment:
Because we are being security tested, I'd suggest moving this username/password into environment variables local to the VM. That way, they don't have to be in the repo.